crates/core/tedge_api/src/health.rs

```diff
@@ -1,25 +1,37 @@
 use crate::mqtt_topics::Channel;
+use crate::mqtt_topics::EntityTopicId;
 use crate::mqtt_topics::MqttSchema;
 use crate::mqtt_topics::ServiceTopicId;
 use clock::Clock;
 use clock::WallClock;
 use log::error;
 use mqtt_channel::MqttMessage;
 use mqtt_channel::Topic;
+use serde::Deserialize;
+use serde::Serialize;
 use serde_json::json;
 use std::process;
 use std::sync::Arc;
 use tedge_utils::timestamp::TimeFormat;
 
-pub const MQTT_BRIDGE_UP_PAYLOAD: &str = "1";
-pub const MQTT_BRIDGE_DOWN_PAYLOAD: &str = "0";
+pub const MOSQUITTO_BRIDGE_PREFIX: &str = "mosquitto-";
+pub const MOSQUITTO_BRIDGE_SUFFIX: &str = "-bridge";
+pub const MOSQUITTO_BRIDGE_UP_PAYLOAD: &str = "1";
+pub const MOSQUITTO_BRIDGE_DOWN_PAYLOAD: &str = "0";
+
 pub const UP_STATUS: &str = "up";
 pub const DOWN_STATUS: &str = "down";
 pub const UNKNOWN_STATUS: &str = "unknown";
```

Comment:
`MOSQUITTO_BRIDGE_UP_PAYLOAD` is only used by a test helper and could be hard-coded there. 

```suggestion
const MOSQUITTO_BRIDGE_UP_PAYLOAD: &str = "1";
const MOSQUITTO_BRIDGE_DOWN_PAYLOAD: &str = "0";

const UP_STATUS: &str = "up";
const DOWN_STATUS: &str = "down";
const UNKNOWN_STATUS: &str = "unknown";
```